### PR TITLE
nf-core modules update -a

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: modules/nf-core/fastqc/main.nf [hunk 1]
```diff
@@ -21,9 +21,15 @@ process FASTQC {
     script:
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
+    // Make list of old name and new name pairs to use for renaming in the bash while loop
+    def old_new_pairs = reads instanceof Path || reads.size() == 1 ? [[ reads, "${prefix}.${reads.extension}" ]] : reads.withIndex().collect { entry, index -> [ entry, "${prefix}_${index + 1}.${entry.extension}" ] }
+    def rename_to = old_new_pairs*.join(' ').join(' ')
+    def renamed_files = old_new_pairs.collect{ old_name, new_name -> new_name }.join(' ')
```
Comment (FriederikeHanssen) [inline modules/nf-core/fastqc/main.nf:27]: complete 🪄 to me

**Review by FriederikeHanssen — COMMENTED** [review]
You split up the test_extended.yml over multiple files?

Table with updated versions to Changelog?

**Comment by maxulysse** [comment]
> You split up the test_extended.yml over multiple files?

Yes, otherwise it takes 45 min and + to completion

> Table with updated versions to Changelog?

I'll check it up rn

**Comment by maxulysse** [comment]
we're done, prettier was done locally, I skipped the CI, because I didn't want to rerun all tests just for a change in CHANGELOG

**Review by FriederikeHanssen — APPROVED** [review]

**Pull request merged**